server/dataone_register.py

```diff
@@ -12,10 +12,7 @@
 
 from girder import logger
 from girder.api.rest import RestException
-
-# http://blog.crossref.org/2015/08/doi-regular-expressions.html
-_DOI_REGEX = re.compile('(10.\d{4,9}/[-._;()/:A-Z0-9]+)', re.IGNORECASE)
```

Comment:
Why'd this go away and get inlined?

Reply:
It was only getting used in one place so I didn't see a reason for it being global. I can move it back if needed.